bin/y-build

```diff
@@ -54,6 +54,17 @@ OUTPUT="type=image,name=$IMAGE,push=true,registry.insecure=true"
 
 BUILD_TAG="$(echo $IMAGE | awk -F/ '{print $NF}' | awk -F@ '{print $1}' | awk -F: '{print $2}')"
 
+# https://github.com/Yolean/build-contract/blob/a7e1a96ccec79bf413d604c344679f7439c34b49/build-contract#L21
+GIT_COMMIT=$(git rev-parse --verify --short HEAD 2>/dev/null || echo '')
+if [[ ! -z "$GIT_COMMIT" ]]; then
+  GIT_STATUS=$(git status --untracked-files=no --porcelain=v2)
+  if [[ ! -z "$GIT_STATUS" ]]; then
+    GIT_COMMIT="$GIT_COMMIT-dev"
+  fi
+  echo "  --- build-contract: This is a git repo and GIT_COMMIT=$GIT_COMMIT ---  "
```

Comment:
Maybe this line is a bit misleading as it mentions build-contract out of nowhere?

Reply:
Also, I was expecting `-dirty` to be appended, and not `-dev`. I'm fine with either, just want to make sure that it's not just something that was copied from build-contract as a mistake